testbed/pyproject.toml

```
@@ -1,8 +1,20 @@
-# This project was generated using template: https://github.com/beeware/briefcase-template and branch: v0.3.12
+[project]
+name = "testbed"
+version = "0.0.1"
+dependencies = ["../core"]
+
+[project.optional-dependencies]
+test = [
+    "coverage==7.5.3",
+    "fonttools==4.53.0",
```

I guess it doesn't matter because this is a testbed project, but this dependency was Android specific. The annoying thing is that we can't use a sys.platform == "android" until at least 3.13; but would it maybe be worth using sys.platform == "linux" so at least the impact of this install is limited to only 2 platforms?
```suggestion
    # fonttools is only needed by Android, but we need to use 
    # sys.platform == 'linux' as there's no dependency identifier
    # that can target Android exclusively until 3.13 lands. 
    "fonttools==4.53.0 ; sys.platform == 'linux'",
```